tests/src/JIT/Performance/CodeQuality/BenchmarksGame/k-nucleotide/k-nucleotide.cs

```diff
@@ -225,7 +225,7 @@ private static void PrepareLookups()
         tochar[3] = 'T';
     }
 
-    [Benchmark]
+    [Benchmark(InnerIterationsCount=Iterations)]
```

Comment:
You'll also want to change the inner for loop to use the InnerIterationsCount:

for(int i=0; i<Benchmark.InnerIterationCount; i++)